private/shell/utils/providers.ts

```diff
@@ -0,0 +1,102 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+
+import { logSpinner } from '.';
+import { outputDirectory } from '../constants';
+
+/**
+ * Get the name of the provider based on the environment variables.
+ *
+ * @returns A string representing the provider name.
+ */
+export const getProvider = (): typeof Bun.env.__BLADE_PROVIDER => {
+  if (Bun.env['CF_PAGES']) return 'cloudflare';
+  if (Bun.env['VERCEL'] === '1') return 'vercel';
+  return '';
+};
+
+/**
+ * Remap inline environment variable definitions
```

Comment:
```suggestion
 * Remap inline environment variable definitions.
```

The other function you added below also had a trailing period, just like the other functions that have TSDoc comments within Blade.